frontend/encore/copy-files.rst

```diff
@@ -54,6 +54,8 @@ include a hash based on their content.
 
 To render inside Twig, use the ``asset()`` function:
 
+.. code-block:: twig
```

Comment:
Should this be `.. code-block:: html+twig` instead?

Reply:
Maybe it should indeed, I'm not used to rst format :-)
Should I make the change?

Reply:
looking at our other code blocks I'd say yes please :)

Reply:
Done, thanks for the insight!